### ISSUE TITLE
[Bug]: Not opening on Mac

### ISSUE THREAD

**Issue by calgaryaereo** [KSP-CKAN/CKAN#4209]
### Is there an existing issue for this?

- [X] I have checked the existing issues for an existing issue. I even actually looked at the ones that are pinned to the top!

### Operating System

macOS Sonoma 14.6.1

### CKAN Version

latest

### Games

KSP 1

### Game Version

latest

### Did you make any manual changes to your game folder (i.e., not via CKAN)?

only adding the 2 DLC's and some .craft files.

### Describe the bug

When clicking on ckan to open, it says cannot open as developer is not trusted. when overrided, app opens for a split second then closes

### Steps to reproduce

Click on app to launch
does not launch

### Relevant log output

```
at System.IO.FileStream..ctor (System.String path, System.IO.FileMode mode, System.IO.FileAccess access, System.IO.FileShare share, System.Int32 bufferSize, System.Boolean anonymous, System.IO.FileOptions options) [0x0019e] in <207d12884485488d9288cc8cbf474289>:0 
  at System.IO.FileStream..ctor (System.String path, System.IO.FileMode mode, System.IO.FileAccess access, System.IO.FileShare share) [0x00000] in <207d12884485488d9288cc8cbf474289>:0 
  at (wrapper remoting-invoke-with-check) System.IO.FileStream..ctor(string,System.IO.FileMode,System.IO.FileAccess,System.IO.FileShare)
  at System.IO.File.OpenRead (System.String path) [0x00000] in <207d12884485488d9288cc8cbf474289>:0 
  at CKAN.SteamLibrary..ctor () [0x00099] in <e7a11ea66e094dd1a3ac6d97032c8cd9>:0 
  at CKAN.GameInstanceManager..ctor (CKAN.IUser user, CKAN.Configuration.IConfiguration configuration) [0x00000] in <e7a11ea66e094dd1a3ac6d97032c8cd9>:0 
  at CKAN.CmdLine.MainClass.Execute (CKAN.GameInstanceManager manager, CKAN.CmdLine.CommonOptions opts, System.String[] args) [0x0028d] in <e7a11ea66e094dd1a3ac6d97032c8cd9>:0 
  at CKAN.CmdLine.MainClass.Main (System.String[] args) [0x000e9] in <e7a11ea66e094dd1a3ac6d97032c8cd9>:0 
```

**Comment by HebaruSan** [comment]
Hi @calgaryaereo, I'm not familiar with this "developer is not trusted" message. Could you please share a screenshot of it so we can start figuring out what it is and what to do about it?

**Comment by calgaryaereo** [comment]
hey @HebaruSan, thanks for getting back to me. the image shows up like this, but ive figured out how to open it. however when I open it there is no window, just the terminal app. when I try to run commands, all I see is that for some reason it is looking in steam library, however I don't even use steam? could you help me figure out how to fix this?
<img width="242" alt="Screenshot 2024-09-28 at 6 49 32 PM" src="https://github.com/user-attachments/assets/dcc684dd-2dbc-4be8-b8a3-0884b5187111">

**Comment by HebaruSan** [comment]
Hi @calgaryaereo, some text is missing at the start of the stack trace in the OP. (It should start with a line that says which exception was thrown.)
If you scroll up, can you find the rest of it?

**Comment by HebaruSan** [comment]
Hi @calgaryaereo, I have added some more fault tolerance to the Steam library loader and switched the Mac image to use the unsigned since apparently it doesn't like our cert anyway. The changes should be available in the dev build for Mac here:

- <https://ksp-ckan.s3-us-west-2.amazonaws.com/CKAN.dmg>

Please give that a try when you have a chance and let us know how it goes.

**Comment by HebaruSan** [comment]
Closing as probably fixed by 00b63219e758acd239c613c67f56db6d89e84b5b and 9134b45ff41cf9e9db21f6ce3e5f971f8b04c191. If this still happens with the above link or the next release, we can re-open this.
